### ISSUE TITLE
Alarms being secretly inactive

### ISSUE THREAD

**Issue by sdasda7777** [qw123wh/Aospdeskclock#25]
Hey, I noticed very peculiar behaviour of alarms. I'm not saying it couldn't be all in my head, but listen to this:
I used this app since the start of this year. Never slept through an alarm. Now on Saturday I turned off all alarms and added a new, later one (not necessarily in this order). On Sunday this alarm worked correctly (at least I think), but later that day I turned it off and turned on the old alarms (again not necessarily in this order). Then on Monday, nothing, not a beep. I thought I must have slept through it, no big deal. However on Tuesday, same thing. Now I was suspicious. I tried moving the last of the active alarms and the inactive alarm to present time to see if it would ring. It did. Then I moved them back. Now today, I was woken up by, wouldn't you know it, the last of the active alarms, the one I moved forth and back.

Basically my hypothesis is that something I did sets alarms as secretly inactive, and moving the alarm can reset this. (All the alarms in question appeared to be active by having the switch blue, which is why I'm saying they were only secretly inactive)

**Comment by sdasda7777** [comment]
Today I got woken up by the first alarm, even though I didn't touch it, so I think I'm most likely mistaken.
